manifest.php

```diff
@@ -40,7 +41,7 @@
         [
             AccessRule::GRANT,
             'http://www.tao.lu/Ontologies/TAO.rdf#PropertyManagerRole',
```

Comment:
Just a suggestion, but I guess you can also replace this value with `TaoRoles` constant.
```suggestion
            TaoRoles::PROPERTY_MANAGER,
```

Reply:
You are right, changed in 808ef6115e4d46b22245c9cfe7e88abd42ec6c45. Thanks!